src/raftstore/store/peer.rs

```diff
@@ -335,6 +344,10 @@ impl Peer {
         let apply_result = try!(self.storage.wl().handle_raft_ready(&ready));
 
         for msg in &ready.messages {
+            if msg.get_msg_type() == MessageType::MsgRequestVote && is_applying {
```

Comment:
is it ok for handing ready when applying?
/cc @xiang90 

Btw, can we need ignoring other peers's message too when applying?